quickwit/quickwit-common/src/metrics.rs

```diff
@@ -252,12 +252,13 @@ impl Default for MemoryMetrics {
 
 #[derive(Clone)]
 pub struct InFlightDataGauges {
-    pub doc_processor_mailbox: IntGauge,
-    pub index_writer: IntGauge,
-    pub indexer_mailbox: IntGauge,
-    pub ingest_router: IntGauge,
     pub rest_server: IntGauge,
+    pub ingest_router: IntGauge,
+    pub wal: IntGauge,
```

Comment:
I reorder stuff in the order of the stream.

Reply:
👍